content/research_report/introductions.md

```diff
@@ -1,61 +1,72 @@
+__Scrubbing__ {cite:p}`power_scrubbing_2012` is a volume censoring approach to remove high motion segments in which the framewise displacement 
+(see section {ref}`framewise-displacement`)
+exceeds some threshold. 
+The scrubbing approach is applied along side head motion parameters and tissue signal regressors.
+__Temporal high-pass filtering__ accounts for low-frequency signal drifts introduced by _physiological and scanner noise sources_.
+Aside from regressors directly modeling noise derived from realignment measures or anatomical properties,
+other approaches capture the impact of motion and non-neuronal physiological activity through data-driven methods. 
+The principle component based method __CompCor__ {cite:p}`behzadi_compcor_2007,muschelli_compcor_2014` extracts principal components from white matter and cerebrospinal fluid masks to estimate non-neuronal activity.
+Independent component analysis based methods estimate spatial independent components representing brain activity and/or noise, 
+and then identify the components related to head-motion through using a data-driven classifier (__ICA-FIX__ {cite:p}`salimi-khorshidi_automatic_2014`)
+or a pre-trained model (__ICA-AROMA__ {cite:p}`aroma`).
 
-These four classes of regressors can be expanded to their first temporal derivatives and their quadratic terms {cite:p}`satterthwaite_2013`.
-The principle component based method __CompCor__ {cite:p}`behzadi_compcor_2007` extracts principal components from white matter and cerebrospinal fluid masks to estimate non-neuronal activity.
-Independent component analysis based methods, __ICA-FIX__ {cite:p}`salimi-khorshidi_automatic_2014` and __ICA-AROMA__ {cite:p}`aroma`,
-estimate independent component time series related to head-motion through a data-driven classifier (ICA-FIX) or a pre-trained model (ICA-AROMA).
-Different strategies have particular strengths,
-and an optimal approach often involves combining a few classes of regressors described above.
+A denoising approach often involves combining a few classes of regressors described above.
+Different strategies have particular strengths and limitations, and researchers can make their own choices based on properties of their data.
+Head motion combied with non-grey matter tissue signal is one of the most basic approach {cite:p}`fox_pnas_2005`. 
+To gain the best denosing results, it often requires the their expansions to module non-linear impact of noise. 
+In addition to the basic parameters, scrubbing has been shown to mitigate the impact of framewise displacement on functional connectome, 
+but removing volumes prevents analysis focusing on frequency characteristics or dynamic changes in fMRI signal, and reduces the temporal degrees of freedom.
+The main argument driving the development of the data-decomposition approach is that the expansions of motion parameters results in high number of regressors,
```

Comment:
It seems like you are switching between two separate issues - scrubbing, and data-decomposition? Is that right? If so I think this paragraph needs to be re-worked a bit. If they are part of the same issue that did not come across for me. Is the link that they are data-driven parameters? If so I think we could frame the paragraph around that

Reply:
Agreed the topic of this paragraph is confusing. L44 and L45 are two different topics. I definitely scrambled too many things in one.
I will extract content related to L44 and expand it to it's own paragraph.
- for each methods, what's the logic of combining things

And then talk about L45:
- each of these methods has their own limitation
- head motion full expansion: lots of noise regressors; overhitting and loss of temporal degrees of freedom
- head motion + scrubbing: loss of temporal degrees of freedom by removing data points
- data driven methods are created to combat the issues above
  - how compcor does it
  - how ica-aroma does it
- problem of data driven methods

WDYT?

Reply:
that sounds much better!

Reply:
Awesome!!!! Thank you so much!!!!